dashboard/pkg/epinio/edit/services.vue

```diff
@@ -60,9 +64,19 @@ export default Vue.extend<Data, EpinioCompRecord, EpinioCompRecord, EpinioCompRe
       errors:                 [],
       failedWaitingForDeploy: false,
       selectedApps:           this.value.boundapps || [],
+      chartValues:            {},
+      validChartValues:       {}
     };
   },
 
+  mounted() {
+    if (this.mode !== 'create') {
+      this.value.details().then((details: any) => {
```

Comment:
See comment for service model `details`. 

This will make a http request to fetch the service that we're already viewing. If the user refreshes on the page it'll be the second request to fetch the same service. 

If there are scenarios where the `settings` section is missing we should make the request optional given that scenario (and use epinio/find with force: true when fetching).

Reply:
Replaced with `store` function.

Reply:
There's still the issue of making an unneeded second request if the user refreshes on the view or edit service page.

![image](https://github.com/epinio/ui/assets/18697775/70f31478-98ac-45e3-a6fd-bc238cacc403)

Even when navigating here from the list page they will already have the resource.

How come we need to make this request, is there a scenario when information is missing?

Reply:
Additional `find` request is done because the custom settings is provided only by `/namespaces/{Namespace}/services` GET request.
Request to add custom settings to the `services` list API: https://github.com/epinio/epinio/issues/2519

Reply:
@richard-cox we verified that the settings are correctly provided by `services` list api, and are provided by `CreateEditView` underneath componente: https://github.com/epinio/epinio/issues/2519#issuecomment-1686602901
Given that, I removed the extra request.